app/src/main/java/eu/kanade/tachiyomi/ui/recent/updates/UpdatesPresenter.kt

```diff
@@ -43,12 +48,19 @@ class UpdatesPresenter(
     private val downloadManager: DownloadManager = Injekt.get(),
     private val getChapter: GetChapter = Injekt.get(),
     preferences: PreferencesHelper = Injekt.get(),
-) : BasePresenter<UpdatesController>(), StableUpdatesState by state {
+) : BasePresenter<UpdatesController>(), UpdatesState by state {
 
     val isDownloadOnly: Boolean by preferences.downloadedOnly().asState()
 
     val isIncognitoMode: Boolean by preferences.incognitoMode().asState()
 
+    val relativeTime: Int by preferences.relativeTime().asState()
+
+    val dateFormat: DateFormat by derivedStateOf { preferences.dateFormat() }
```

Comment:
You will also need to add the import for this

```suggestion
    val dateFormat: DateFormat by mutableStateOf(preferences.dateFormat())
```